lib/plugins/helper/toc.js

```diff
@@ -21,8 +21,14 @@ function tocHelper(str, options = {}) {
   let lastLevel = 0;
 
   headings.each(function() {
+    function getId(ele) {
```

Comment:
Please bring this `getId` function above `headings.each(...)` scope.

Reply:
I see. I'll push tweaks later

Reply:
Thanks for your patience with us